packages/web-pkg/src/composables/sideBar/eventTopics.ts

```diff
@@ -0,0 +1,7 @@
+export abstract class SideBarEventTopics {
```

Comment:
Should be very straightforward to convert this to a string-enum, right? Even without code changes in the consuming code locations.